flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/StreamTwoInputSelectableProcessor.java

```diff
@@ -264,46 +284,55 @@ private boolean waitForAvailableInput(InputSelection inputSelection)
 		throws ExecutionException, InterruptedException, IOException {
```

Comment:
nit: could you add a java doc above `waitForAvailableInput` explaining what's the meaning of return value? Maybe something like:
```
@return false if both of the inputs are finished true otherwise
```
?

Reply:
Javadoc has been added, and the PR has been updated.